index.html

```diff
@@ -841,8 +842,73 @@ <h2>
                 determines if and when the cursor is included in the captured display surface.
               </p>
             </dd>
+            <dt>
+               <dfn>restrictOwnAudio</dfn> of type {{boolean}}
+            </dt>
+            <dd>
+              <p>
+                Indicates whether the <a href="#dfn-restrictownaudio">restrictOwnAudio</a>
+                constraint is applied (<code>true</code>) or not (<code>false</code>).
+              </p>
+            </dd>
           </dl>
         </section>
+        <section>
+          <h2>
+            Extensions to {{MediaTrackCapabilities}}
+          </h2>
+          <p>
+            When the {{MediaStreamTrack/getCapabilities()}} method is invoked on a video stream track,
+            the user agent must return the extended {{MediaTrackCapabilities}} dictionary,
+            representing the capabilities of the underlying user agent.
+          </p>
+          <pre class="idl"
+>partial dictionary MediaTrackCapabilities {
+  DOMString displaySurface;
+  boolean logicalSurface;
+  sequence&lt;DOMString&gt; cursor;
+  boolean restrictOwnAudio;
+};</pre>
+          <dl data-link-for="MediaTrackCapabilities" data-dfn-for="MediaTrackCapabilities" class=
+          "dictionary-members">
+            <dt>
+              <dfn>displaySurface</dfn> of type {{DOMString}}
+            </dt>
+            <dd>
+              <p>
+                MUST be the same value as is returned by {{MediaStreamTrack/getSettings()}},
+                rendering this property immutable from the application's viewpoint.
+              </p>
+            </dd>
+            <dt>
+              <dfn>logicalSurface</dfn> of type {{boolean}}
+            </dt>
+            <dd>
+              <p>
+                MUST be the same value as is returned by {{MediaStreamTrack/getSettings()}},
+                rendering this property immutable from the application's viewpoint.
+              </p>
+            </dd>
+            <dt>
+               <dfn>cursor</dfn> of type sequence&lt;DOMString&gt;
+            </dt>
+            <dd>
+              <p>
+                MUST consist of exactly the set of values from {{CursorCaptureConstraint}}
+                that the user agent is capable of supporting for this track.
+              </p>
+            </dd>
+          <dt>
+             <dfn>restrictOwnAudio</dfn> of type {{boolean}}
+          </dt>
+          <dd>
+            <p>
+              MUST be the same value as is returned by {{MediaStreamTrack/getSettings()}},
+              rendering this property immutable from the application's viewpoint.
```

Comment:
let's make this settable; if we can change it on the fly, return [true false]; if we can't, return one value.

Reply:
Filed #180 for this.